challenge_3/python/returnlove/src/majority_element.py

```diff
@@ -0,0 +1,26 @@
+def find_majority_element(input_list):
+	majority_condition = len(input_list)/2
+	# print(majority_condition)	
+	my_dict = {}
+	# print(input_list)
+	# convert input list into dictionary, key being the element and value is the count
+	for element in input_list:
```

Comment:
Something cool to look at is something called `defaultdict(int)`. If you try to access something not in the dictionary, it'll automatically give that value a 0.